rllib/policy/dynamic_tf_policy.py

```diff
@@ -679,7 +679,8 @@ def _initialize_loss_from_dummy_batch(
                         key not in [
                             SampleBatch.EPS_ID, SampleBatch.AGENT_INDEX,
                             SampleBatch.UNROLL_ID, SampleBatch.DONES,
-                            SampleBatch.REWARDS, SampleBatch.INFOS]:
+                            SampleBatch.REWARDS, SampleBatch.INFOS,
+                            SampleBatch.OBS_EMBEDS]:
```

Comment:
No need to do this rn, but in a follow up PR, we should also test-run the callbacks, so that the SampleBatch access detector captures this field and automatically adds it to the view-requirements (instead of always doing so).